### DISCUSSION TITLE
How to make GlazeWM launch when windows starts?

### DISCUSSION THREAD

**Discussion by OmarGonD** [glzr-io/glazewm#716] [Q&A]
I want GlazeWM to launch on windows start. How to do it?

**Comment by OmarGonD** [comment]
You have to put the shortcut of GlazeWM here: C:\Users\omago\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup

**Comment by scifisatan** [comment]
thank you so much

**Comment by AndreEriksenNLTG** [comment]
Startup is delayed in windows 11 - furthermore the right path is shell:startup 
press WIN+R (Run dialog) 
shell:startup
Make shortcut there
